docs/components/sensor/_index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Sensor Component"
+linkTitle: "Sensor"
+weight: 70
+draft: false
+type: "docs"
+description: "A device that sends information about the outside world to the computer controlling a robot."
+tags: ["sensor", "components"]
+icon: "img/components/sensor.png"
```

Comment:
```suggestion
no_list: true
icon: "img/components/sensor.png"
```

Reply:
We don't have the list of cards at the end of the other components

Reply:
sounds good, done!